LOS_Interface/Inc/config.hpp

```diff
@@ -11,5 +11,26 @@ const uint8_t GIMBAL_GRABBER_TOGGLE_INDEX = 5; // three-way switch for gimbal, g
 const uint8_t LEFT_GIMBAL_GRABBER_CRANE = 6;
 const uint8_t RIGHT_GIMBAL_GRABBER_MOUTH = 7;
 
+/* LOS_Pos */
+#include "imu.hpp"
+#include "BMX160.hpp"
+#include "MPU6050.hpp"
+
+#include "airspeed.hpp"
+#include "MPXV7002DP.hpp"
+
+#include "altimeter.hpp"
+#include "MPL3115A2.hpp"
+
+#include "gps.hpp"
+#include "NEOM8.hpp"
```

Comment:
so if we do not have all of these connected, would we comment out the includes ?

Is there an easy way to say #define <SENSOR> [0/1] that tells us if it's there or not?

Reply:
Really sorry for the late reply, but yeah I think it might be better to use define guards i think, should I try using them instead

Reply:
Or not define guards sorry like using #ifdef I am not sure what to call them sorry about that

Reply:
They''re called "preprocessor directive", if you could have them in and get the PR merged soon that would be epic!

Reply:
Thanks, I just added them in and commit the changes. Also for the overall comment I wasn't sure how to reply to it but for testing on hardware I haven't been able to do so in the past week since I have been busy with some course work but I am planning on testing within this text week